Tests/JsonApiSerializerTests.cs

```diff
@@ -83,6 +83,22 @@ public void UsesQueryFieldsetExpressions()
             Assert.NotNull(result["data"][0]["attributes"]["location"]);
             Assert.Null(result["data"][0]["attributes"]["number-of-employees"]);
         }
+        
+        [Fact(DisplayName = "Uses query fieldset expressions also on relationships if specified")]
+        public void UsesQueryFieldsetExpressionsOnRelationships()
+        {
+            var target = new JsonApiSerializer<PersonResource>
+            {
+                AllowQuery = true
+            };
+            var people = Get.People(1).ToList();
+            var result = target.Serialize(people, new Uri(DefaultUrl, "?fields[person]=Job"));
+            _output.WriteLine(result.ToString());
+
+            var relationships = result["data"][0]["relationships"];
+
+            Assert.Null(relationships["family-members"]);
```

Comment:
Should you also assert that `Job` actually is there?

Reply:
I should and I will :)